zds/middlewares/matomomiddleware.py

```diff
@@ -88,7 +104,21 @@ def process_response(self, request, response):
                 if request.path.startswith(p):
                     return response
             try:
-                self.matomo_track(request)
+                if (
+                    p.contains(reverse("search:query"))
```

Comment:
```suggestion
                    reverse("search:query") in p
```

Sinon on se prend un `AttributeError 'str' object has no attribute 'contains'`.

Reply:
c'est modifié.